src/nanoemoji/nanoemoji.py

```diff
@@ -270,16 +272,30 @@ def _draw_glyph_extents(ufo: ufoLib2.Font, glyph: Glyph):
 
 
 def _glyf_ufo(ufo, color_glyphs):
+    # glyphs by reuse_key
+    glyphs = {}
+    swaps = []
```

Comment:
this `swaps` local variable seems unused

Reply:
ha I think I thought I'd swap something. Fixed!